Server/src/main/java/com/bbd/BeanServer/controller/PostController.java

```diff
@@ -70,7 +83,64 @@ public ResponseEntity<Void> postReaction(@RequestBody Map<String, Object> reques
         return ResponseEntity.ok().build();
     }
 
+    @PostMapping("/posts")
+    ResponseEntity<?> returnAllPosts(@RequestBody String request) {
+        return ResponseEntity.ok(postRepository.findAll());
+    }
+
+    @PostMapping("/posts/user")
+    ResponseEntity<?> returnUserPosts(@RequestBody Users user) {
+        Optional<Users> actualUser = userService.getUserByName(user.getUsername());
+
+        if (actualUser.isPresent()) {
+            long id = actualUser.get().getUser_id();
+
+            List<Post> userPosts = postRepository.findAll().stream()
+                    .filter(post -> post.getUserId() == id)
+                    .collect(Collectors.toList());
+
+            return ResponseEntity.ok(userPosts);
+        } else {
+            return ResponseEntity.status(HttpStatus.NOT_FOUND).body(null);
+        }
+    }
+
+    @PostMapping("/posts/new")
+    ResponseEntity<?> returnNewestPost(@RequestBody String mystring) {
+
```

Comment:
same here, get request since youre not creating anything

Reply:
It's just the way my template function works atm, might change later.